demo/src/Img2Vis/index.tsx

```diff
@@ -36,8 +36,7 @@ const Img2VisPanel = ({ width }: { width: string | number }) => {
       decSecretJSON.constructor === Object &&
       decSecretJSON !== undefined &&
       // eslint-disable-next-line no-prototype-builtins
-      decSecretJSON.hasOwnProperty('data') &&
-      decSecretJSON.hasOwnProperty('config')
+      decSecretJSON.hasOwnProperty('data') && decSecretJSON.hasOwnProperty('config')
```

Comment:
could just use `decSecretJSON.data && decSecretJSON.config` for values checking. 

or you could also use `Object.keys(decSecretJSON).includes('data', 'config')` for keys checking

try~